### PR TITLE
correction of alias to generate php-symfony

### PR DESCRIPTION
**Pull request by HichemChiouchiou**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`~~, `3.4.x`, `4.0.x`~~. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

The alias of the Symfony's Bundle who is generate by Openapi was open_apiserver. However, it doesn't work it missing an underscore between api and server. The good alias is open_api_server.

### CODE REVIEW

**Comment by oneslash** [comment]
cc: @jebentier, @dkarlovi, @mandrean, @jfastnacht, @ackintosh, @ybelenko, @renepardon

**Comment by oneslash** [comment]
cc: @ksm2

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java [hunk 1]
```diff
@@ -244,6 +249,12 @@ public void processOpts() {
             additionalProperties.put(BUNDLE_NAME, bundleName);
         }
 
+        if (additionalProperties.containsKey(BUNDLE_ALIAS)) {
+            this.setBundleAlias((String) additionalProperties.get(BUNDLE_ALIAS));
+        } else {
+            additionalProperties.put(BUNDLE_ALIAS, bundleAlias);
```
Comment (wing328) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java]: @HichemChiouchiou  If BundleAlias is not provided, what about defaulting to the logic in line 193 to make the fix backward compatible?
```
        this.bundleAlias = snakeCase(bundleName).replaceAll("([A-Z]+)", "\\_$1").toLowerCase(Locale.ROOT);
```

**Comment by wing328** [comment]
For consistency, what about adding an option similar to https://github.com/OpenAPITools/openapi-generator/blob/b0552205f319c52869f6833b567dbf909ba825e9/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java#L179 ?

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java (hunk 1 above)
Comment (HichemChiouchiou) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java]: Done!

**Comment by oneslash** [comment]
@wing328, we have fixed comments :)

**Pull request merged**